public/components/integrations/components/setup_integration.tsx

```diff
@@ -400,15 +486,15 @@ const addIntegration = async ({
       sessionId = result.value.sessionId ?? sessionId;
     }
     // Once everything is ready, add the integration to the new datasource as usual
-    // TODO determine actual values here after more about queries is known
     const res = await addIntegrationRequest(
       false,
       integration.name,
       config.displayName,
       integration,
       setCalloutLikeToast,
       config.displayName,
-      `flint_${config.connectionDataSource}_default_${config.connectionTableName}_mview`
+      `flint_${config.connectionDataSource}_default_${config.connectionTableName}_mview`,
+      config.enabledWorkflows
```

Comment:
Intentionally only enabling conditional workflows for S3 integrations and not all usages of `addIntegrationRequest`